### DISCUSSION TITLE
Enforcement no permissionamento de nós

### DISCUSSION THREAD

**Discussion by gabrielsdev** [bacen/pilotord-kit-onboarding#29] [Q&A] [answered]
Olá, pessoal,

Gostaria de saber se haverá algum tipo de enforcement no permissionamento de nós da rede. 

Caso um nó esteja listado no smart contract de permissionamento de nós mas não habilite o permissionamento de nó no config.toml, este nó pode se conectar com um outro fora da lista de permissionamento e, assim, permitir que nós despermissionados participem da rede.

Desse modo, será desenvolvido algum tipo de reforço para o permissionamento ou as configurações de firewall e permissionamento são consideradas suficientes?

Obrigado, desde já!

**Comment by aldenio** [comment, marked as answer]
Prezado @gabrielsdev ,

Estamos cientes da situação relativa ao permissionamento via arquivo config.toml e  estamos considerando o uso de permissionamento por smart contract, permissionamento das wallets, o uso de TLS e firewalls como mecanismos de proteção, além da monitoração ativa dos nós da rede. Nossa equipe técnica ainda não chegou a um consenso se os mecanismos aplicados são ou não suficientes para garantir o correto funcionamento da rede.


ps: Caso alguém tenha sugestões adicionais para essa situação, estamos abertos a ouvir.  
ps: Vale ressaltar que estamos tratando do projeto **piloto** do real digital, e que uma de suas funções é identificar pontos de atenção para que possamos mapear controles que terão que ser implementados quando passarmos a desenvolver o sistema que irá para produção.
